src/apps/lwaftr/lwaftr.lua

```diff
@@ -307,13 +307,30 @@ local function cannot_fragment_df_packet_error(lwstate, pkt)
                                  ethernet_header_size, icmp_config)
 end
 
--- Given a packet containing IPv4 and Ethernet, encapsulate the IPv4 portion.
-local function ipv6_encapsulate(lwstate, pkt, next_hdr_type, ipv6_src, ipv6_dst,
-                                 ether_src, ether_dst)
-   -- TODO: decrement the IPv4 ttl as this is part of forwarding
-   -- TODO: do not encapsulate if ttl was already 0; send icmp
+local function encapsulate_and_transmit(lwstate, pkt, ipv6_dst, ipv6_src)
+   -- Do not encapsulate packets that now have a ttl of zero or wrapped around
+   local ttl = decrement_ttl(pkt)
+   if ttl == 0 or ttl == 255 then
```

Comment:
When TTL is zero a ICMPv4 packet with message "ttl exceed in transist" is sent unless policy is DROP. My question is when TTL can be 255 and if that's value is related with ```default_ttl``` defined in ```constants.lua```? (```default_ttl``` is also 255).